core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/toolbar/withUVEToolbar.ts

```diff
@@ -37,11 +37,12 @@ import { EditorToolbarState, PersonaSelectorProps, UVEToolbarProps } from '../mo
  * @property {boolean} initialState.isPreviewModeActive - Flag indicating whether the preview mode is active.
  */
 const initialState: EditorToolbarState = {
-    device: DEFAULT_DEVICES.find((device) => device.inode === 'default'),
```

Comment:
@kevindaviladev @zJaaal If we need to access a default item from an array in multiple places, we should create a separate `default` variable/constant and add it to the array. This way, we can reference the `default` item directly without having to iterate over the array each time.